index.js

```diff
@@ -222,9 +222,15 @@ class TagInput extends Component {
   };
 
   removeIndex = (index: number) => {
-    const tags = [...this.props.value];
+    const tags = _.clone(this.props.value);
```

Comment:
![simulator screen shot 5 jun 2017 13 48 49](https://cloud.githubusercontent.com/assets/2046717/26781161/c660b032-49f5-11e7-8cd0-3ead061cd15a.png)
Getting error "_ is not defined".
With ```[...this.props.value];``` everything works as expected.